lib/index.js

```diff
@@ -21,6 +21,10 @@ const cache = require('./util/cache');
  * main class
  */
 class Mcdev {
+    static buObject = {};
+    static properties;
+    static cred;
+    static bu;
```

Comment:
I would need to either pass properties in parameter to another function, or have 
const properties = await config.getProperties(); twice, so thought it would be easier to record it in static class variable. Same for buObject, cred and bu (otherwise need to pass that as parameters)